src/app/loadout-builder/generated-sets/FilterBuilds.tsx

```diff
@@ -8,6 +8,17 @@ import { D2ManifestDefinitions } from 'app/destiny2/d2-definitions';
 import styles from './FilterBuilds.m.scss';
 import { statHashes, statKeys } from '../process';
 import { statTier } from './utils';
+import { useDispatch } from 'react-redux';
+import { setSetting } from 'app/settings/actions';
+
+const statTypeToHash: { [type in StatTypes]: number } = {
+  Mobility: 2996146975,
+  Resilience: 392767087,
+  Recovery: 1943323491,
+  Discipline: 1735777505,
+  Intellect: 144602215,
+  Strength: 4244567218,
```

Comment:
We probably only need one of these in the app, and can produce the inverse at runtime.

Reply:
Sure, will sort it out in my next PR.